app/components/sessions/session-agenda-nav.hbs

```diff
@@ -1,15 +1,25 @@
-<ul class="vlc-tabs-reverse" role="tablist">
-  <li class="vlc-tabs-reverse__item">
+<ul class="auk-tabs auk-tabs--reversed" role="tablist">
+  <li class="auk-tabs__tab auk-tabs__hierarchical-back">
+    <LinkTo @route="agendas.overview">
+      <span class="auk-tabs__tab-inner">
+        <div class="auk-icon auk-icon--hierarchical-back auk-icon--default">
+          <i class="ki ki-hierarchical-back"></i>
+        </div>
+      </span>
+    </LinkTo>
+  </li>
```

Comment:
constructies zoals deze, is er een vervolg story voorzien om ook hier auComponenten te gebruiken?
in de au-tab bestaat een if voor deze

```
{{#if @isHierarchicalBack}}
  <li class="auk-tabs__tab auk-tabs__hierarchical-back" ...attributes>
    <LinkTo @route={{this.route}}>
      <span class="auk-tabs__tab-inner">
        <WebComponents::AuTab::Innerlayout @layout={{this.layout}} @icon="hierarchical-back" @counter={{this.counter}}>
          {{yield}}
        </WebComponents::AuTab::Innerlayout>
      </span>
    </LinkTo>
  </li>
{{else}}
```

Reply:
Correct! Deze is voorzien om achteraf een sweep te doen om deze soort herhalingen er uit te halen.

(Vooral om scope beperkt te houden, sneller te kunnen vooruitgang boeken en nu nog niet te moeten inzitten met eventuele issues door de refactor naar componenten.)

Reply:
cava, dan comment ik dit soort zaken niet in deze PR :)